databricks/koalas/groupby.py

```diff
@@ -84,16 +84,25 @@ class GroupBy(object, metaclass=ABCMeta):
     :type _groupkeys: List[Series]
     """
 
-    def __init__(self):
-        """A constructor for attribute type hints."""
-        self._kser = cast(Series, None)
-        self._kdf = cast(DataFrame, None)
-        self._groupkeys = cast(List[Series], None)
-        self._as_index = cast(bool, None)
-        self._dropna = cast(bool, None)
-        self._column_labels_to_exlcude = cast(Set[Tuple], None)
-        self._agg_columns_selected = cast(bool, None)
-        self._agg_columns = cast(List, None)
+    def __init__(
+        self,
+        _kser: Series = None,
+        _kdf: DataFrame = None,
+        _groupkeys: List[Series] = None,
+        _as_index: bool = False,
+        _dropna: bool = False,
+        _column_labels_to_exlcude: Set[Tuple] = None,
+        _agg_columns_selected: bool = False,
+        _agg_columns: List = None,
+    ):
```

Comment:
nit: Maybe we don't need to use underscore for name of arguments in general ??

```diff
-        _kser: Series = None,
-        _kdf: DataFrame = None,
-        _groupkeys: List[Series] = None,
-        _as_index: bool = False,
-        _dropna: bool = False,
-        _column_labels_to_exlcude: Set[Tuple] = None,
-        _agg_columns_selected: bool = False,
-        _agg_columns: List = None,
+        kser: Series = None,
+        kdf: DataFrame = None,
+        groupkeys: List[Series] = None,
+        as_index: bool = False,
+        dropna: bool = False,
+        column_labels_to_exlcude: Set[Tuple] = None,
+        agg_columns_selected: bool = False,
+        agg_columns: List = None,

```

Reply:
Sounds good! Renamed without underscores.